TeXmacs/doc/main/links/man-index.zh.tm

```diff
@@ -0,0 +1,32 @@
+<TeXmacs|2.1.1>
+
+<style|<tuple|tmdoc|old-spacing|old-dots|old-lengths|chinese>>
+
+<\body>
+  <tmdoc-title|\<#751F\>\<#6210\>\<#7D22\>\<#5F15\>>
+
+  \<#4E3A\>\<#4E86\>\<#751F\>\<#6210\>\<#7D22\>\<#5F15\>\<#FF0C\>\<#5728\>\<#7B2C\>\<#4E00\>\<#9636\>\<#6BB5\>\<#FF0C\>\<#4F60\>\<#5FC5\>\<#987B\>\<#4F7F\>\<#7528\><menu|Insert|Link|Index
+  entry>\<#5C06\>\<#7D22\>\<#5F15\>\<#9879\>\<#653E\>\<#5165\>\<#6587\>\<#6863\>\<#4E2D\>\<#3002\>\<#5728\>\<#7B2C\>\<#4E8C\>\<#9636\>\<#6BB5\>\<#FF0C\>\<#4F60\>\<#5FC5\>\<#987B\>\<#5C06\>\<#5149\>\<#6807\>\<#653E\>\<#5728\>\<#8981\>\<#751F\>\<#6210\>\<#7D22\>\<#5F15\>\<#7684\>\<#4F4D\>\<#7F6E\>\<#FF0C\>\<#7136\>\<#540E\>\<#5355\>\<#51FB\><menu|Insert|Automatic|Index>\<#3002\>\<#7136\>\<#540E\>\<#4EE5\>\<#4E0E\>\<#76EE\>\<#5F55\>\<#7C7B\>\<#4F3C\>\<#7684\>\<#65B9\>\<#5F0F\>\<#751F\>\<#6210\>\<#7D22\>\<#5F15\>\<#3002\>
+
+  \<#5728\>\<#63D2\>\<#4EF6\>\<rightarrow\>\<#94FE\>\<#63A5\>\<rightarrow\>\<#7D22\>\<#5F15\>\<#9879\>\<#83DC\>\<#5355\>\<#4E2D\>\<#FF0C\>\<#60A8\>\<#53EF\>\<#4EE5\>\<#627E\>\<#5230\>\<#51E0\>\<#79CD\>\<#7C7B\>\<#578B\>\<#7684\>\<#7D22\>\<#5F15\>\<#9879\>\<#3002\>\<#6700\>\<#7B80\>\<#5355\>\<#7684\>\<#662F\>\Pmain\Q\<#3001\>\Psub\Q\<#3001\>\Psubsub\Q\<#FF0C\>\<#5B83\>\<#4EEC\>\<#662F\>\<#5206\>\<#522B\>\<#6709\>\<#4E00\>\<#4E2A\>\<#3001\>\<#4E24\>\<#4E2A\>\<#548C\>\<#4E09\>\<#4E2A\>\<#53C2\>\<#6570\>\<#7684\>\<#5B8F\>\<#3002\>
```

Comment:
> 插件→链接→索引项

Like the first paragraph, you should use `<menu|Insert|Link|Index entry>` instead.

Reply:
fix the problem now